nautobot/extras/tests/test_views.py

```diff
@@ -1904,7 +1904,6 @@ class GraphQLQueriesTestCase(
     ViewTestCases.GetObjectViewTestCase,
     ViewTestCases.GetObjectChangelogViewTestCase,
     ViewTestCases.ListObjectsViewTestCase,
-    ViewTestCases.BulkEditObjectsViewTestCase,
```

Comment:
Hm, it'd be better to keep this mixin and have the test exercise something that can be bulk-edited, e.g. a custom field.

Reply:
It does not show up in custom fields content types, I tend to agree

<img width="1191" height="205" alt="image" src="https://github.com/user-attachments/assets/41de9270-75b1-4eca-9ccf-8fd29fb14027" />